cmd/incus/storage_bucket.go

```diff
@@ -3,13 +3,17 @@ package main
 import (
```

Comment:
Another comment on this one, the changes to po/ should be in a separate commit as `i18n: Update translation templates`